i18n/ko/campaigns/tcu/disappearance_at_the_twilight_estate.po

```diff
@@ -448,3 +448,6 @@ msgstr "'페니의 운명'을 기록하기"
 
 msgid "Would you like to skip the prologue by drawing tarot cards?"
 msgstr "타로 카드 뽑기를 통해 서막을 건너뛰기 하시겠습니까? (‘돌아온 황혼회 저택에서의 실종’에서 추가된 선택 규칙입니다.)"
+
+msgid "Do you want to use the app to draw random tarot cards?"
+msgstr "앱을 통해 무작위로 타로 카드를 뽑으시겠습니까?"
```

Comment:
Thanks for flagging this, it actually goes in other project but I'll make sure this translation makes it over there.